providers/vm_extension_removal_policy.rb

```diff
@@ -0,0 +1,68 @@
+
+require 'json'
+
+action :set do
+  begin
+    # get the path of extension config file
+    if node['platform_family'] == 'windows'
+      extension_path = Dir["C:/Packages/Plugins/Chef.Bootstrap.WindowsAzure.ChefClient/*"].last

```

Comment:
Need to get these paths from the manifest, we can't assume we know what they are. Or if we don't go from the manifest, we at least need to use environment variables, i.e. we can't assume c: even exists, but we could use `systemdrive`.